pkgs/record_use/example/record_use_example.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:record_use/record_use.dart';
+
+void doStuff(RecordedUsages usage, Identifier callId, Identifier referenceId) {
```

Comment:
Do you want to provide an example of using `RecordedUsages.fromFile`? And / or borrow the main example from the readme.

Reply:
I think `RecordedUsages.fromFile` is an internal method/implementation detail, the users will get a deserialized `RecordUsages` object in their `link.dart`.